src/main/java/teammates/ui/output/AccountRequestData.java

```diff
@@ -0,0 +1,52 @@
+package teammates.ui.output;
+
+import java.time.Instant;
+
+import teammates.common.datatransfer.attributes.AccountRequestAttributes;
+
+/**
+ * Output format of account request data.
+ */
+public class AccountRequestData extends ApiOutput {
+
+    private final String email;
+    private final String name;
+    private final String institute;
+    private final String registrationKey;
+    private final Instant registeredAt;
+    private final Instant createdAt;
```

Comment:
As API output, these should be `Long`. `Instant` is only relevant within the realms of Java.